src/ocamlorg_frontend/components/learn_components.eml

```diff
@@ -154,31 +154,46 @@ let right_books_content_block books ~title ~(link : link) =
   </div>
 
 let contribute_footer ~href =
-  <div class="pt-10 mt-16 border-t border-slate-200 lg:grid lg:grid-cols-3 gap-4 text-slate-500">
-    <div class="lg:col-span-1">
-      <h2 class="mt-0 mb-4 text-base font-semibold leading-6 lg:text-base">Help Improve Our Documentation</h2>
-      <p class="text-sm mt-0 mb-4 max-w-xs">All OCaml docs are open source. See something that's wrong or unclear? Submit a pull request.</p>
-      <a
-        class="inline-block relative py-1 px-4 font-medium no-underline align-middle bg-transparent rounded-md border border-solid appearance-none cursor-pointer select-none whitespace-no-wrap hover:no-underline"
-        href="<%s href %>"
-      >
-        <%s! Icons.github_pull_request "inline-block overflow-visible mr-0 align-text-bottom select-none" %>
-        Contribute</a
-      >
+  <div class="my-5 border-y border-slate-400 pt-5">
```

Comment:
Please use the same border color as other existing borders on the site. This will simplify tidying up the colors when the dark mode project starts.

We will have color variables in the `tailwind.config.js` then and it will be easier to rename existing colors when we have more consistency between them.

Here, I recommend `border-gray-200`. (Please change all the borders.)